pkg/varlinkapi/containers.go

```diff
@@ -47,7 +47,7 @@ func (i *LibpodAPI) ListContainers(call iopodman.VarlinkCall) error {
 func (i *LibpodAPI) GetContainer(call iopodman.VarlinkCall, id string) error {
 	ctr, err := i.Runtime.LookupContainer(id)
 	if err != nil {
-		return call.ReplyContainerNotFound(id)
+		return call.ReplyContainerNotFound(id, err.Error())
```

Comment:
Could you just pass in error and then everyone would not need to call Error()?

Reply:
nope, doesnt work that way given the errors are typed as structures.